storage/aptosdb/src/pruner/worker.rs

```diff
@@ -129,5 +134,7 @@ impl Worker {
 
 pub enum Command {
     Quit,
-    Prune { target_db_versions: Vec<Version> },
+    Prune {
+        target_db_versions: Vec<Option<Version>>,
```

Comment:
nit: let's document what the first and second elements are.

Or we can make pruning ledger store and pruning state store to be separate commands, so we can have the ability to have more precise control on how frequently we want to run them in the future.

Reply:
Added some documentation for now. Will separate the command into 2 in a separate PR if needed.